Api/Models/Restaurant.cs

```diff
@@ -83,6 +83,11 @@ public class Restaurant
     [StringLength(50)]
     public required string IdCardFileName { get; set; }
 
+    /// <summary>
+    /// Max reservation duration
+    /// </summary>
+    public required int MaxReservationDuration { get; set; }
```

Comment:
Lepiej podpisać że to minuty
```suggestion
    public required int MaxReservationDurationMinutes { get; set; }
```